app/experimenter/experiments/tests/test_forms.py

```diff
@@ -1284,15 +1309,41 @@ def test_optional_reviews(self):
 
         self.assertEqual(
             form.optional_reviews,
-            (
+            [
+                form["review_advisory"],
                 form["review_legal"],
                 form["review_ux"],
                 form["review_security"],
                 form["review_vp"],
                 form["review_data_steward"],
                 form["review_comms"],
                 form["review_impacted_teams"],
-            ),
+            ],
+        )
+
+    def test_optional_reviews_when_a_risk_option_is_true(self):
+        experiment = ExperimentFactory.create_with_status(
+            Experiment.STATUS_REVIEW,
+            review_relman=True,
+            review_science=True,
+            risk_partner_related=True,
+        )
+
+        form = ExperimentReviewForm(
+            request=self.request, data={}, instance=experiment
+        )
+
```

Comment:
I would just check that the 'review_vp' moved from the optional list to the required list here:

    self.assertIn(form["review_vp"], form.required_reviews)
    self.assertNotIn(form["review_vp"], form.optional_reviews)

The other test above already checks that the whole optional list looks as expected so you don't need to check the whole thing again.